client/blocks/app-promo/qr-code.jsx

```diff
@@ -0,0 +1,35 @@
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { QRCodeSVG } from 'qrcode.react';
+
+import './style.scss';
+
+export const QrCode = ( { source = 'calypso-customer-home', size = 150 } ) => {
+	const translate = useTranslate();
+	const localizeUrl = useLocalizeUrl();
+	return (
+		<div className="app-promo__qr-code">
+			<QRCodeSVG
+				value={ localizeUrl( `https://apps.wordpress.com/get?campaign=${ source }` ) }
```

Comment:
Could be useful to add tracking that they came via QR code — you'd see difference between clicks and QR code reads.

Typically I see [UTM codes](https://en.wikipedia.org/wiki/UTM_parameters) used with marketing pages but seems like you have `campaign` already set up.